helm/charts/hpe-csi-driver/templates/hpe-csi-controller.yaml

```diff
@@ -128,9 +131,7 @@ spec:
             - "--flavor=kubernetes"
             - "--pod-monitor"
             - "--pod-monitor-interval=30"
-            {{- if .Values.cspClientTimeout }}
-            - "--csp-client-timeout={{ .Values.cspClientTimeout }}s"
-            {{- end }}
+            - "--csp-client-timeout=360s"
```

Comment:
I don't think we wanted to increase the timeout for all CSPs here. We wanted to get rid of this and override to 360 only for 3par/Primera CSP using the[ hack](https://github.com/hpe-storage/csi-driver/blob/ae0812a550cd3e569fef38444e4818e28a624473/pkg/driver/driver.go#L362) here instead of setting it globally for all [csps](https://github.com/hpe-storage/csi-driver/blob/ae0812a550cd3e569fef38444e4818e28a624473/pkg/driver/driver.go#L74)